warrior/Actions/CommonActions/common_actions.py

```diff
@@ -118,7 +118,7 @@ def verify_resp_data(self, resp_ref, resp_pat, object_key):
         return status
 
     def store_in_repo(self, datavar=None, datavalue=None, type='str',
-                      filepath=None, jsonkey="repo_variables"):
+                      filepath=None, jsonkey="repo_variables", slurp=False):
```

Comment:
The argument name ```slurp``` is cute, I must admit, but it does not convey the true meaning of that argument (hence, there is another meaning to slurp that I am not aware of). I will strongly recommend changing it something more meaningful.

Reply:
please suggest one, can i use 'store_all_from_file'

Reply:
used argument name bool_store_all